composer.json

```diff
@@ -18,7 +18,7 @@
         }
     ],
     "require": {
-        "yiisoft/yii2": ">=2.0.4",
+        "yiisoft/yii2": "*",
```

Comment:
I think it is good to require `>2.0.4` to indicate dev dependency for something after 2.0.4